components/TwoslashPatch.tsx

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from "react";
+import { createPortal } from "react-dom";
+import {useCustomStyle} from '@/hooks/useCustomStyle'
+
+export default function TwoslashPatchPortal() {
+
+    const {
+        isMounted,
+        currentStyle
+    } = useCustomStyle()
+
+    return  <>
+        {isMounted &&
```

Comment:
the initial render is the mount, this conditional is redundant. It can only reach this line to check `isMounted` _if it's mounted_. This boolean is actually saying `isTheSecondRenderOrLater`

Reply:
change the name "isMounted" to "isClientReady"

I don't want to use "useState" for checking is SSR or CSR. I did something various but It is the only way to prevent the error `Error: Hydration failed because the initial UI does not match what was rendered on the server.`